application/event_replay/event_replay.py

```diff
@@ -0,0 +1,95 @@
+from decimal import Decimal
+from os import getenv
+from time import time_ns
+from typing import Any, Dict
+
+from aws_lambda_powertools import Logger, Tracer
+from aws_lambda_powertools.utilities.typing import LambdaContext
+from boto3 import client
+from boto3.dynamodb.types import TypeDeserializer
+from common.middlewares import unhandled_exception_logging
+from simplejson import dumps
+
+tracer = Tracer()
+logger = Logger()
+
+
+@tracer.capture_lambda_handler()
+@unhandled_exception_logging
+@logger.inject_lambda_context
+def lambda_handler(event: Dict[str, Any], context: LambdaContext) -> str:
+    """Entrypoint handler for the authoriser lambda
+
+    Args:
+        event (Dict[str, Any]): Lambda function invocation event
+        context (LambdaContext): Lambda function context object
+
+    Returns:
+        dict: Policy to allow connection to the API Gateway Mock
+    """
+    correlation_id = build_correlation_id()
```

Comment:
do we need to build a new correlation_id?

Reply:
hmm interesting question? i think we probably do want a new one but might be worth giving the option of the caller of the lambda to pass one in the event body then we can find it in logs, with current implementation it would be more fiddly as we would need to work out what time_ns() was or do a more fuzzy query based on env-replayed-event

Reply:
@rohith1122 The correlation id is new so that it only relates to the replayed event and not the original